contentService/src/main/java/com/kovan/app/controller/BucketController.java

```diff
@@ -0,0 +1,49 @@
+package com.kovan.app.controller;
+
+import com.kovan.app.service.BucketService;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.IOException;
+
+@RestController
+@RequestMapping("/bucket")
+public class BucketController {
```

Comment:
what do you mean by Bucket here ? change it to S3controller and mapping as well

Reply:
changed to S3Controller and mapping as well.